opencti-platform/opencti-graphql/src/migrations/1687082905415-types-adapt.js

```diff
@@ -7,7 +7,7 @@ const entityTypeChange = (fromType, toType, indices) => {
   const updateQuery = {
     script: {
       params: { toType, prefix: convertTypeToStixType(toType) },
-      source: "ctx._source.entity_type = params.toType; ctx._source.standard_id = (params.prefix + '--' + ctx._source.standard_id.splitOnToken('--')[1]);",
+      source: 'ctx._source.entity_type = params.toType;',
```

Comment:
Can you explain this changes ?

Reply:
We don't want to modify the standard_id, just the entity_type because at the end we want our Threat-Actor-Group to be just a Threat-Actor for stix